index.html

```diff
@@ -5,8 +5,143 @@
+                <img src="#" class="photo-right" alt="Здание в Седоне">
+              </li>
+              <li class="advantages-item">
+                <img src="#" class="icon">
+                <h3 class="advantages-black">Жильё</h3>
+                <p class="advantages-description">Рекомендуем пожить в настоящем мотеле, всё как в кино!</p>
+              </li>
+              <li class="advantages-item">
+                <img src="#" class="icon" 
+                <h3 class="advantages-black">Еда</h3>
+                <p class="advantages-description">Всегда заказывайте топовый фирменный бургер, вы не разочаруетесь!</p>
+              </li>
+              <li class="advantages-item">
+                <img src="#" class="icon">
+                <h3 class="advantages-black">Сувениры</h3>
+                <p class="advantages-description">Не только китайского, но и настоящего местного производства!</p>
+              </li>
+              <li class="advantages-item">
+                <img src="#" class="photo-left" alt="Мост Дьявола">
+                <h3 class="advantages-white">Там есть мост дьявола</h3>
+                <p class="advantages-number">-№2-</p>
+                <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+              </li>
+              <li class="advantages-item">
+                <h3 class="advantages-black">Небольшая площадь</h3>
+                <p class="advantages-number">-3-</p>
+                <p class="advantages-description">Все достопримечательности находятся очень близко</p>
+              </li>
+              <li class="advantages-item">
+                <h3 class="advantages-black">Красивая дорога</h3>
+                <p class="advantages-number">-№4-</p>
+                <p class="advantages-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+              </li>
+              <li class="advantages-item">
+                <h3 class="advantages-black">Мало туристов</h3>
+                <p class="advantages-number">-№5-</p>
+                <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+              </li>
+            </ul>         
+        </section>  
+
+        <article class="modal">
+          <h2 class="title">Заинтересовались?</h2>
+          <p class="search-hotel">Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+          <input type="submit" value="ПОИСК ГОСТИНИЦЫ В СЕДОНЕ">
+        </article>
+
+        <article class="subscribes">
+          <h2 class="subtitle">Подпишитесь на рассылку</h2>
+          <p>Только полезная информация и никакого спама, честное бойскаутское!</p>
+          <form action="#" method="post">
+            <input type="text" name="email" value="Ваш e-mail">
+            <input type="submit" value="Подписаться">
+          </form>
+        </article>
```

Comment:
- В принципе большинство тут тоже делали бы секцию. Но не то, чтобы `article` совсем не подходит
- Для всех инпутов нужны лейблы. Можно сделать скрытый или использовать тот текст что уже есть
- Лучше использовать `button`
- `type="email"
`
Либо можно весь блок сделать одной формой.
```suggestion
    <form class="subscribes" action="#" method="post">
      <h2 class="subtitle">
        <label for="subscribe">Подпишитесь на рассылку</label>
      </h2>
      <p>Только полезная информация и никакого спама, честное бойскаутское!</p>

      <input type="email" id="subscribe" name="email" placeholder="Ваш e-mail" autocomplete="email">
      <button>Подписаться</button>
    </form>
```